js/widgets/page.js

```diff
@@ -39,83 +40,66 @@ $.widget( "mobile.page", {
 		theme: "a",
 		domCache: false,
 
-		// Deprecated in 1.4 remove in 1.5
-		keepNativeDefault: $.mobile.keepNative,
-
-		// Deprecated in 1.4 remove in 1.5
-		contentTheme: null,
 		enhanceWithin: true,
 		enhanced: false
 	},
 
-	// DEPRECATED for > 1.4
-	// TODO remove at 1.5
-	_createWidget: function() {
-		$.Widget.prototype._createWidget.apply( this, arguments );
-		this._trigger( "init" );
-	},
-
 	_create: function() {
+
 		// If false is returned by the callbacks do not create the page
 		if ( this._trigger( "beforecreate" ) === false ) {
 			return false;
 		}
 
-		if ( !this.options.enhanced ) {
-			this._enhance();
-		}
-
-		this._on( this.element, {
-			pagebeforehide: "removeContainerBackground",
-			pagebeforeshow: "_handlePageBeforeShow"
-		} );
+		this
```

Comment:
No need to chain all these and make them all return as a general rule we dont chain widget methods and they dont return anything unless there is a good reason.